packages/apollo-gateway/src/index.ts

```diff
@@ -59,6 +59,7 @@ interface GatewayConfigBase {
   experimental_didUpdateComposition?: Experimental_DidUpdateCompositionCallback;
   experimental_pollInterval?: number;
   experimental_approximateQueryPlanStoreMiB?: number;
+  experimental_compressDownstreamRequests?: boolean;
```

Comment:
This sounds a bit like an option that would enable Gzip/Deflate.  Could we call it, idk,  `experimental_autoFragmentization`?

Reply:
Yeah, I'm good with that.
98bdb30